src/test/java/org/folio/fqm/utils/SqlFieldIdentificationUtilsTest.java

```diff
@@ -10,25 +10,59 @@
 class SqlFieldIdentificationUtilsTest {
 
   @Test
-  void testRmbJsonbColumn() {
+  void testRmbJsonbResultsColumn() {
     ColumnValueGetter columnValueGetter = new ColumnValueGetter()
       .type(ColumnValueGetter.TypeEnum.RMB_JSONB)
       .param("name");
 
     EntityTypeColumn entityTypeColumn = new EntityTypeColumn()
       .name("derived_column_name_01")
       .valueGetter(columnValueGetter);
-    var actualField = SqlFieldIdentificationUtils.getSqlField(entityTypeColumn);
+    var actualField = SqlFieldIdentificationUtils.getSqlResultsField(entityTypeColumn);
     var expectedField = field("jsonb ->> 'name'");
     assertEquals(expectedField, actualField);
   }
 
   @Test
-  void testSqlFieldWhenValueGetterMissing() {
+  void testSqlResultsFieldWhenValueGetterMissing() {
     EntityTypeColumn entityTypeColumn = new EntityTypeColumn()
       .name("derived_column_name_01");
-    var actualField = SqlFieldIdentificationUtils.getSqlField(entityTypeColumn);
+    var actualField = SqlFieldIdentificationUtils.getSqlResultsField(entityTypeColumn);
     var expectedField = field(entityTypeColumn.getName());
     assertEquals(expectedField, actualField);
   }
+
+  @Test
+  void testRmbJsonbFilterColumn() {
+    ColumnValueGetter columnValueGetter = new ColumnValueGetter()
+      .type(ColumnValueGetter.TypeEnum.RMB_JSONB)
+      .param("name");
+
+    EntityTypeColumn entityTypeColumn = new EntityTypeColumn()
+      .name("derived_column_name_01")
+      .filterValueGetter("this is a filter which is being overridden by the value getter")
+      .valueGetter(columnValueGetter);
+    var actualField = SqlFieldIdentificationUtils.getSqlFilterField(entityTypeColumn);
+    var expectedField = field("jsonb ->> 'name'");
+    assertEquals(expectedField, actualField);
+  }
+
+  @Test
+  void testSqlFilterFieldWhenValueGetterMissing() {
+    EntityTypeColumn entityTypeColumn = new EntityTypeColumn()
+      .name("derived_column_name_01");
+    var actualField = SqlFieldIdentificationUtils.getSqlFilterField(entityTypeColumn);
+    var expectedField = field(entityTypeColumn.getName());
+    assertEquals(expectedField, actualField);
+  }
```

Comment:
Hopefully, Sonar doesn't get too upset that I pretty much just duplicated the 2 existing tests and changed 1 line in each...